### ISSUE TITLE
PHP standards

### ISSUE THREAD

**Issue by kdauzickas** [bobthecow/genghis#27]
Genghis is failing on servers where php warnings/notices are displayed. Call to /geghis.php/servers spews out this:

```
Notice: Undefined index: databases in genghis.php on line 14

Warning: array_map(): Argument #2 should be an array in genghis.php on line 14

Notice: Undefined index: totalSize in genghis.php on line 14
```

Usually this is harmless but because the js part is expecting JSON these messages brake the script.

Possible sollutions: following strict standards or error_reporting(0);

**Comment by bobthecow** [comment]
I use strict standards :)

I've seen this bug before, but only as a side effect of #16. Genghis doesn't handle non-privileged accounts very well. This will be fixed in v2.1.

**Comment by bobthecow** [comment]
Closing as a dupe of #16. If this isn't the case, feel free to reopen.
